GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
-* Scheduling discussing of proposals that have been submitted.
-* Asking for new proposals to be made to address an identified need.
-* Oversee disciplinary action for members. The Chairs have the authority to declare a member inactive and expel members for cause.
-* Chairs will serve for one-year revolving terms and will be approved using the [Condorcet Method](https://en.wikipedia.org/wiki/Condorcet_method). Upon the expiration of a Chair’s term, it will continue for another year, unless challenged.
-
-The terms for founding Chairs start on the approval of this charter.
+## Mission
 
-When no candidate has submitted their name for consideration, the current Chairs may appoint an acting Chair until a candidate comes forward.
+The mission of the Operator Framework community is:
+* To collaborate on areas related to developing, packaging, deploying
+    and managing the lifecycle of Kubernetes Operators
 
-Chairs must be active members. Any inactivity, inability to filfill the role, or ineligibility results in immediate removal.
+* To develop informational resources that give the community an
+    understanding of best practices, trade-offs, and value adds as it
+    relates to developing, packaging, deploying and managing the
+    lifecycle of Kubernetes Operators
 
-Chairs may be removed by petition to the Operator Framework Community through the Call for Agreement process outlined above.
+* To identify suitable projects and gaps in the ecosystem for future
+    collaboration and coordination with those projects.
 
-Additional Chairs may be added so long as the existing number of Chairs is odd. These Chairs are added using a Call for Agreement. Extra Chairs enjoy the same rights, responsibilities, and obligations of a Chartered Chair. Upon vacancy of an Extra Chair, it may be filled by appointment by the remaining Chairs, or a majority vote of the Operator Framework Community until the term naturally expires.
+### Scope
 
-In the event that an even number of Chairs exist and vote situation arises, the Chairs will randomly select one chair to abstain. 
+The Operator Framework community focuses on the following
+developer-related topics of creating Kubernetes Operators:
+* Code scaffolding to create Kubernetes Operators
+* Libraries supporting common needs of Operator authors
+* Packaging of Kubernetes Operators
+* Testing of Kubernetes Operators
 
-[OWNERS_ALIASES]: https://github.com/operator-framework/community/blob/master/OWNERS_ALIASES
+The Operator Framework community focuses on the following admin-related
+topics of running Kubernetes Operators on a Kubernetes cluster:
+* Maintaining catalogs of Operators
+* Discovering and installing Operators from those catalogs
+* Managing the lifecycle of continuous updates and eventual removal of Operators
+
+The Operator Framework community will work on developing best practices,
```

Comment:
The of community develops best practices, fosters collaboration, and improves tool interoperability